packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -109,14 +109,19 @@ export default function ShareOptionsMenu({
             <MenuIcon Icon={DefaultSquadIcon} />
           ),
           text: `Share to ${squad.name}`,
-          action: () =>
+          action: () => {
+            trackEvent({ event_name: AnalyticsEvent.StartShareToSquad });
```

Comment:
This is a post event and should include all the properties we include in other post events (see for example click, or upvote post)